### PR TITLE
docs: update challenge

### PR DESCRIPTION
**Pull request by felipefialho**
- Update image of layout
- [Add link of Figma](https://www.figma.com/file/ED8oK1CyH2BD5BjSAcy9vF/Teste-Front-end?node-id=505%3A2613)
- Update text
- Change some rules

### CODE REVIEW

**Review by hick97 — APPROVED** [review]

**Review by HelenDias — APPROVED** [review]
🚀

**Review by JulioCVaz — APPROVED** [review]
🚀

File: README.md [hunk 1]
```diff
@@ -20,58 +20,43 @@ The main objective of this challenge is to test your skills in Front-end Develop
 
 ## The Challenge
 
-The [following layout](layout.jpg) need to be developed.
+### About layout
```
Comment (ipeternella) [inline README.md]: **suggestion (non-blocking):** 
```suggestion
### About the layout
```

File: README.md [hunk 2]
```diff
@@ -20,58 +20,43 @@ The main objective of this challenge is to test your skills in Front-end Develop
...
 
-<img src="layout.jpg" width="800">
+The [following Figma layout needs to be developed](https://www.figma.com/file/ED8oK1CyH2BD5BjSAcy9vF/Teste-Front-end?node-id=505%3A2613).
 
-It's important to say that it is just a prototype, the idea is to check your ability to propose improvements, features and contribute to the product UI 😄. Feel free to make improvements like new functionalities, animations, visual optimizations, etc.
+<img src="layout-jsm.png" width="800">
+
+It's important to say that it is just a prototype, the idea is to check your ability to propose improvements, features and contribute to the product UI 😄.
 
 The logo is the same as the one on the repository.
```
Comment (ipeternella) [inline README.md]: **suggestion (non-blocking):** my friend, what do you think of the following structure:

```suggestion
The logo is the same as the one in the repository.

It's important to say that it is just a prototype! We would also like to see your ability to propose improvements, features and new contributions to the product's UI 😄!
```

File: README.md [hunk 3]
```diff
@@ -20,58 +20,43 @@ The main objective of this challenge is to test your skills in Front-end Develop
...
 
-You should develop:
+**Feel completely free** to make improvements like:
```
Comment (ipeternella) [inline README.md]: **suggestion (non-blocking):** 
```suggestion
**Feel completely free** to make improvements such as:

- Adding new functionalities;
- Creating animations;
- Developing visual optimizations;
- etc.
```

File: README.md [hunk 4]
```diff
@@ -20,58 +20,43 @@ The main objective of this challenge is to test your skills in Front-end Develop
...
+### About API
 
 The JSON that you need to develop for this application is available at:
 
 - `https://jsm-challenges.s3.amazonaws.com/frontend-challenge.json`
 
-To optimize our service, we divided our consultants into three regions. Consider the points below to perform the filters.
-
-- **ESPECIAL**
+You can feel free to use BFF (Back-end for Front-end) before using it on client-side, if you think that if it makes sense 👀
```
Comment (ipeternella) [inline README.md]: **suggestion (non-blocking):**
```suggestion
Feel free to use BFF (Back-end for Front-end) before using it on the client-side, if you think that if it makes sense 👀!
```

File: README.md [hunk 5]
```diff
@@ -86,4 +71,4 @@ After finishing the test, send an email to vagas-dev@juntossomosmais.com.br, wit
 
 If you have any questions or suggestions, send your questions directly to us (vagas-dev@juntossomosmais.com.br) or open an issue.
 
-Please, check if your question has already been answered in [a issue](https://github.com/juntossomosmais/frontend-challenge/issues?q=) :relaxed:
+Please, check if your question has already been answered in [an issue](https://github.com/juntossomosmais/frontend-challenge/issues?q=) 😌
```
Comment (IgooorGP) [inline README.md:74]: **nitpick:** 
```suggestion
Please, check if your question has already been answered on [an issue](https://github.com/juntossomosmais/frontend-challenge/issues?q=) 😌
```

**Review by ipeternella — COMMENTED** [review]
Kudos for the improvements, my friend!! 🎉 🎉 🎉

**Comment by felipefialho** [comment]
Thanks a lot @IgooorGP ❤️

File: README.md [hunk 6]
```diff
@@ -20,58 +20,46 @@ The main objective of this challenge is to test your skills in Front-end Develop
...
+- Adding new functionalities;
+- Creating animations;
+- Developing visual optimizations;
+- etc.
 
-- `https://jsm-challenges.s3.amazonaws.com/frontend-challenge.json`
+- New functionalities
+- Animations
+- Visual optimizations
+- etc.
```
Comment (ipeternella) [inline README.md]: **issue:** isn't this part being repeated?

File: README.md [hunk 7]
```diff
@@ -86,4 +74,4 @@ After finishing the test, send an email to vagas-dev@juntossomosmais.com.br, wit
 
 If you have any questions or suggestions, send your questions directly to us (vagas-dev@juntossomosmais.com.br) or open an issue.
```
Comment (ipeternella) [inline README.md]: **suggestion:** 
```suggestion
If you have any questions or suggestions, send your questions directly to us (vagas-dev@juntossomosmais.com.br) or open an issue. But, before that, please, make sure that your question hasn't been answered on a previous [issue](https://github.com/juntossomosmais/frontend-challenge/issues?q=)
```

**Review by ipeternella — COMMENTED** [review]
Awesome work!! Just one last question and suggestion lol

**Review by ipeternella — APPROVED** [review]
🚀 🚀 🚀 !! You rock, my friend! Nice job, @felipefialho !!

**Pull request merged**